### DISCUSSION TITLE
Link needs updated

### DISCUSSION THREAD

**Discussion by MrBrain295** [GoogleChrome/lighthouse#14126] [General] [answered]
The link in the GitHub description needs to be updated to https://developer.chrome.com/docs/lighthouse/overview/.

**Comment by adamraine** [comment]
What are you referring to? That is already the link we use.

**Comment by brendankenny** [comment, marked as answer]
I think maybe it's the link in the "About" section on https://github.com/GoogleChrome/lighthouse. The old link redirects there anyways, but I updated it to point directly there now. Good eye @MrBrain295!
